internal/pkg/deploy/cloudformation/stack/transformers.go

```diff
@@ -13,10 +14,42 @@ import (
 	"github.com/aws/copilot-cli/internal/pkg/template"
 )
 
+const (
+	enabled  = "ENABLED"
+	disabled = "DISABLED"
+)
+
+// Default values for EFS options
+const (
+	defaultRootDirectory   = "/"
+	defaultIAM             = disabled
+	defaultReadOnly        = true
+	defaultWritePermission = false
+)
+
+var (
+	pDefaultReadOnly      = aws.Bool(defaultReadOnly)
+	pDefaultIAM           = aws.String(defaultIAM)
+	pDefaultRootDirectory = aws.String(defaultRootDirectory)
+	pEnabled              = aws.String(enabled)
```

Comment:
Im still a bit worried about those pointer variables that are potentially used in multiple places. The reason is because
```go
a := pDefaultIAM
``` 
if we accidentally modify the value of `*a` then the value of `*pDefaultIAM` will get changed as well, which is not what we expect and will make it hard to debug.

Reply:
Good point. I've gotten rid of the pointers entirely and just called `aws.String` or `aws.Bool` on the consts where appropriate.

Reply:
Sounds good! Feel free to remove the label.